packages/block-library/src/comments-pagination-next/edit.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { useBlockProps, PlainText } from '@wordpress/block-editor';
+
+const arrowMap = {
+	none: '',
+	arrow: '→',
+	chevron: '»',
+};
+
+export default function CommentsPaginationNextEdit( {
+	attributes: { label },
+	setAttributes,
+	context: { paginationArrow },
+} ) {
+	const displayArrow = arrowMap[ paginationArrow ] || arrowMap.none;
+	return (
+		<div { ...useBlockProps() }>
+			<a
+				href="#comments-pagination-next-pseudo-link"
+				onClick={ ( event ) => event.preventDefault() }
+			>
+				<PlainText
+					__experimentalVersion={ 2 }
+					tagName="span"
+					aria-label={ __( 'Next comments page link' ) }
+					placeholder={ __( 'Next Comments' ) }
+					value={ label }
+					onChange={ ( newLabel ) =>
+						setAttributes( { label: newLabel } )
+					}
+				/>
+				{ displayArrow && (
+					<span className={ `wp-block-query-pagination-next-arrow` }>
```

Comment:
It looks like a copy and paste issue:

```suggestion
					<span className="wp-block-comments-pagination-next-arrow">
```

Reply:
It is not 😅 . The function that renders the arrow has that class hardcoded, so, in order to be the same both in editor and frontend and also with the same styles of the query pagination, should include that name.